cmd/archive_upload_client/client.go

```diff
@@ -22,6 +22,12 @@ import (
 	grpcMetadata "google.golang.org/grpc/metadata"
 )
 
+const (
+	// Max message size set to 10mb.
+	// Max message size set to 10mb.
+	maxMsgSize = 10 * 1024 * 1024
```

Comment:
RouteViews has some update archives over 10 MiB: http://routeviews.org/route-views3/bgpdata/2021.12/UPDATES/ 

Maybe 50 is a good line?

Another thing: will we ever consider to import RV RIBs? Those can be pretty large (i.e. over 100 MiBs), but I guess we can worry about those later.

Reply:
I think setting to 50m sounds fine, we can cross the RIB part later yea.